pkg/applicationserver/io/packages/alcsync/v1/package.go

```diff
@@ -73,9 +75,27 @@ func (a *alcsyncpkg) HandleUp(
 		return nil
 	}
 
-	commands, err := MakeCommands(msg, fPort, data)
+	eventBuilders := make(events.Builders, 0)
```

Comment:
I think you should defer this before merging the application data, such that errors coming from that generate an event too.